amplify/backend/backend-config.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
what was this change

Reply:
this should be empty since there is no backend configured yet